code/modules/mob/living/carbon/human/species_types/mothmen.dm

```diff
@@ -36,7 +36,34 @@
 		H.adjustToxLoss(3)
 		H.reagents.remove_reagent(chem.type, REAGENTS_METABOLISM)
 
+/datum/reagent/toxin/mothbgon/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
+	if(!ismoth(M))
+		return
+
+	var/mob/living/carbon/victim = M
+	if(method == TOUCH || method == VAPOR)
+		//check for protection
+		//actually handle the pepperspray effects
+		if(!victim.is_eyes_covered() || !victim.is_mouth_covered())
+			victim.blur_eyes(2)
+			victim.blind_eyes(2) // 6 seconds
+			victim.Knockdown(1 SECONDS)
+			victim.emote("scream")
+			victim.emote("spin")
+			victim.adjustToxLoss(8)
+			victim.adjustOxyLoss(8)
+			victim.confused = max(M.confused, 10) // 10 seconds
+			victim.add_movespeed_modifier(MOVESPEED_ID_PEPPER_SPRAY, update=TRUE, priority=100, multiplicative_slowdown=0.25, blacklisted_movetypes=(FLYING|FLOATING))
+			addtimer(CALLBACK(victim, /mob.proc/remove_movespeed_modifier, MOVESPEED_ID_PEPPER_SPRAY), 10 SECONDS)
+		victim.update_damage_hud()
+
+
 /datum/species/moth/check_species_weakness(obj/item/weapon, mob/living/attacker)
 	if(istype(weapon, /obj/item/melee/flyswatter))
 		return 9 //flyswatters deal 10x damage to moths
 	return 0
+
+/datum/species/moth/check_species_weakness(obj/item/weapon, mob/living/attacker)
+	if(istype(weapon, /obj/item/melee/flyswatter/moth))
+		return 29 //flyswatters deal 30x damage to moths
```

Comment:
same here, what does 29 mean

Reply:
Bonus damage

Reply:
this should be in mothswatter not in species